1.7/administration/high-availability/mult-zone.md

```diff
@@ -0,0 +1,41 @@
+---
+post_title: Multiple Zones
+menu_order: 1
+---
+
+Here are recommendations for specific multi-AZ configurations.
+
+**Important:** A typical DC/OS cluster has all master and agent nodes in the same zone. The cost of having masters spread across zones usually outweigh the benefits. None of the following multi-AZ setups have been explicitly tested or verified by Mesosphere.
```

Comment:
This is not strictly true right? We have tested setup (2) with our soak cluster now.